src/test/java/uk/ac/cam/cl/dtg/segue/comm/EmailManagerTest.java

```diff
@@ -436,32 +427,27 @@ public final void sendRegistrationConfirmation_checkForInvalidTemplateTags_throw
 
     } catch (ContentManagerException e) {
       e.printStackTrace();
-      Assert.fail();
+      Assertions.fail();
     }
 
     EmailManager manager = new EmailManager(emailCommunicator, userPreferenceManager, mockPropertiesLoader,
         mockContentManager, logManager, generateGlobalTokenMap());
-    try {
-      ImmutableMap<String, Object> emailTokens = ImmutableMap.of("verificationURL", "https://testUrl.com");
 
+    ImmutableMap<String, Object> emailTokens = ImmutableMap.of("verificationURL", "https://testUrl.com");
```

Comment:
I know you're just copying the previous code, but we shouldn't be using `ImmutableMap` when we can simply just use `Map`:

```java
Map<String, Object> emailTokens = Map.of("verificationURL", "https://testUrl.com");
```

This does the same thing, and it's part of the core library, where `ImmutableMap` is from Google. We should prioritise using the standard library over third party libraries, unless we have a good reason not to. For instance, readability would be a good reason, but I think in this case they're bot readable.